internal/core/core.go

```diff
@@ -557,3 +557,22 @@ func StripQuotes(name string) string {
 	}
 	return name
 }
+
+func GetPathsBasePath(options *CompilerOptions, currentDirectory string) string {
```

Comment:
This could just be a method on `CompilerOptions`, I think.

Reply:
It's already a private function in `module`, could you remove it from there if you make it a CompilerOptions method?